themes/doubletime/doubletime.theme.bash

```diff
@@ -28,45 +28,16 @@ else
     THEME_PROMPT_HOST_COLOR="$blue"
 fi
 
-doubletime_scm_prompt() {
-  CHAR=$(scm_char)
-  if [ $CHAR = $SCM_NONE_CHAR ]; then
-    return
-  elif [ $CHAR = $SCM_GIT_CHAR ]; then
-    echo "$(git_prompt_status)"
-  else
-    echo "[$(scm_prompt_info)]"
-  fi
-}
-
 function prompt_setter() {
   # Save history
   history -a
   history -c
   history -r
   PS1="
 $(clock_prompt) $(scm_char) [${THEME_PROMPT_HOST_COLOR}\u@${THEME_PROMPT_HOST}$reset_color] $(virtualenv_prompt)$(ruby_version_prompt)\w
-$(doubletime_scm_prompt)$reset_color $ "
+$(scm_prompt)$reset_color $ "
   PS2='> '
   PS4='+ '
 }
 
 safe_append_prompt_command prompt_setter
-
-git_prompt_status() {
-  local git_status_output
-  git_status_output=$(git status 2> /dev/null )
-  if [ -n "$(echo $git_status_output | grep 'Changes not staged')" ]; then
-    git_status="${bold_red}$(scm_prompt_info) ✗"
-  elif [ -n "$(echo $git_status_output | grep 'Changes to be committed')" ]; then
-     git_status="${bold_yellow}$(scm_prompt_info) ^"
-  elif [ -n "$(echo $git_status_output | grep 'Untracked files')" ]; then
-     git_status="${bold_cyan}$(scm_prompt_info) +"
-  elif [ -n "$(echo $git_status_output | grep 'nothing to commit')" ]; then
-     git_status="${bold_green}$(scm_prompt_info) ${green}✓"
```

Comment:
We're effectively dropping this color support for this theme. Are we cool with that?

Reply:
I am pretty okay with that, thanks for letting me know though :smile: